developer-docs/upgrade/index.md

```diff
@@ -17,27 +17,57 @@ This page details out the jobs required to be run as part of the upgrade from Su
 |-------------|------------|--------|
 |dock_private_ingressgateway_ip|Knowledge Platform FlinkJobs|Update in KP/common.yml<br/>Add dock env private ingress ip|
 |learning_content_drive_apiKey||Update in Core/common.yml<br/>[Create the Key](https://support.google.com/googleapi/answer/6158862?hl=en) and Add in Sunbird and VDN|
+|metadata2_redis_host|KP Flink|Same value as it is in DP common.yaml<br/>In KP hosts.yaml, add redis-metadata2, copy this from DP hosts.yaml|
+|crypto_encryption_key_external|Portal|generate encryption key using [link](https://www.allkeysgenerator.com/Random/Security-Encryption-Key-Generator.aspx)|
+|adminutil__portal_keyprefix: "portalv2_key"|adminutil|add in core/common.yml|
```

Comment:
@G33tha The implementation has changed in 4.3.0. So these are not relavent. Please remove these - Line # 22 to Line # 47.

Reply:
abobe variables are removed